components/dbutil/src/main/java/com/wso2telco/core/dbutils/DbService.java

```diff
@@ -379,37 +379,25 @@ public boolean UpdateApplicationOp(int appID, int operatorid, boolean opstat) th
      */
     public Integer applicationEntry(int appID, Integer[] operators) throws Exception {
 
-        Connection con = null;
-        PreparedStatement statement = null;
         Integer newid = 0;
 
-        try {
-            con = DbUtils.getDBConnection();
-            if (con == null) {
-                throw new Exception("Connection not found");
-            }
-
-            for (Integer d : operators) {
-                StringBuilder queryString = new StringBuilder("INSERT INTO ");
-                queryString.append("operatorapps ");
-                queryString.append("(applicationid,operatorid) ");
-                queryString.append("VALUES (?, ? ) ");
-
-                statement = con.prepareStatement(queryString.toString());
+        for (Integer d : operators) {
+            StringBuilder queryString = new StringBuilder("INSERT INTO ");
+            queryString.append("operatorapps ");
+            queryString.append("(applicationid,operatorid) ");
+            queryString.append("VALUES (?, ? ) ");
 
+            try (Connection con = DbUtils.getDBConnection();
+                 PreparedStatement statement = con.prepareStatement(queryString.toString())) {
```

Comment:
This will make DbUtils.getDBConnection() to be called for each iteration of  for (Integer d : operators)

Reply:
I will change it.